tools/Modules/TestFx-Tasks.psd1

```diff
@@ -0,0 +1,126 @@
+#
+# Module manifest for module 'PSGet_TestFx-Tasks'
+#
+# Generated by: ShahAbhijeet
+#
+# Generated on: 11/18/2016
+#
+
+@{
+
+# Script module or binary module file associated with this manifest.
```

Comment:
I'm not sure I understand this script.  Where is the issue that prompted this work, and what do we want this to do?  Is it just about making the Azure cmdlets easier to use?